j2/rules/webiny_risk.yaml

```diff
@@ -0,0 +1,123 @@
+# DO NOT UPDATE THIS FILE MANUALLY; ANY CHANGES WILL BE OVERRIDDEN BY THE J2 DIR TEMPLATES
+updatereq:
+  vertices:
+    
+    - label: Object_Data_Source
+      name: webiny
+      props:
+        - name: Object_Data_Source_Name
+          val: WEBINY/RISCOS-DE-FONTES-DE-DADOS
+          mandatoryInSearch: True
+    
+    - label: Object_Data_Source
+      name: attached_data_sources
+      condition: ${riscosDeFontesDeDadosFontesDeDados?.size() > 0}
+      props:
+        - name: Object_Data_Source_Form_Id
+          val: ${new groovy.json.JsonBuilder(riscosDeFontesDeDadosFontesDeDados)}
+          mandatoryInSearch: True
+          processAll: True
+          type: "[Lcom.pontusvision.utils.WebinyRef;"
+          postProcessor: ${com.pontusvision.utils.WebinyRef.fromString(it,'id')}
+    
+    - label: Object_Risk_Data_Source
+      props:
+        - name: Object_Risk_Data_Source_Form_Id
+          val: ${id}
+          mandatoryInSearch: True
+        - name: Object_Risk_Data_Source_Risk_Id
+          val: ${riscosDeFontesDeDadosReferencia?.trim()?.toUpperCase()}
+          mandatoryInSearch: False
+          excludeFromSearch: True
+        - name: Object_Risk_Data_Source_Description
+          val: ${riscosDeFontesDeDadosDescricao?.trim()?.toUpperCase()}
+          mandatoryInSearch: False
+          excludeFromSearch: True
+        - name: Object_Risk_Data_Source_Probability
+          val: ${riscosDeFontesDeDadosProbabilidade}
+          mandatoryInSearch: False
+          excludeFromSearch: True
+        - name: Object_Risk_Data_Source_Impact
+          val: ${riscosDeFontesDeDadosImpacto}
+          mandatoryInSearch: False
+          excludeFromSearch: True
+        - name: Object_Risk_Data_Source_Residual_Probability
+          val: ${riscosDeFontesDeDadosProbResidual}
+          mandatoryInSearch: False
+          excludeFromSearch: True
+        - name: Object_Risk_Data_Source_Residual_Impact
+          val: ${riscosDeFontesDeDadosImpResidual}
+          mandatoryInSearch: False
+          excludeFromSearch: True
+        - name: Object_Risk_Data_Source_Approved_By_DPO
```

Comment:
Please simplify to true

Reply:
next PR i'll do that ... when all POLEs are done!